eras/alonzo/test-suite/test/Test/Cardano/Ledger/Alonzo/Serialisation/Tripping.hs

```diff
@@ -8,160 +8,155 @@
 
 module Test.Cardano.Ledger.Alonzo.Serialisation.Tripping where
 
-import Cardano.Binary
-import Cardano.Ledger.Alonzo (AlonzoEra)
+import Cardano.Ledger.Alonzo (Alonzo)
 import Cardano.Ledger.Alonzo.Data (BinaryData, Data (..))
-import qualified Cardano.Ledger.Alonzo.Data as Data
 import Cardano.Ledger.Alonzo.Rules (AlonzoUtxoPredFailure, AlonzoUtxosPredFailure, AlonzoUtxowPredFailure)
 import Cardano.Ledger.Alonzo.Scripts (CostModels)
-import qualified Cardano.Ledger.Alonzo.Scripts as Script
-import Cardano.Ledger.Alonzo.TxBody (txBodyRawEq)
 import Cardano.Ledger.Alonzo.TxWits (AlonzoTxWits)
 import Cardano.Ledger.Block (Block)
 import Cardano.Ledger.Core
 import Cardano.Ledger.Shelley.Metadata (ShelleyTxAuxData)
 import Cardano.Protocol.TPraos.BHeader (BHeader)
-import Codec.CBOR.Write (toLazyByteString)
-import qualified Data.ByteString.Base16.Lazy as Base16
-import qualified Data.ByteString.Lazy.Char8 as BSL
-import Data.Functor.Identity (Identity (..))
-import Data.Roundtrip (roundTrip, roundTripAnn, roundTripAnnWithTwiddling, roundTripWithTwiddling)
-import Data.Twiddle (Twiddle (..))
 import Test.Cardano.Ledger.Alonzo.Serialisation.Generators ()
+import Test.Cardano.Ledger.Binary.Arbitrary ()
+import Test.Cardano.Ledger.Binary.RoundTrip
+-- import Test.Cardano.Ledger.Binary.Twiddle (Twiddle (..))
 import Test.Cardano.Ledger.Shelley.ConcreteCryptoTypes
 import Test.Cardano.Ledger.ShelleyMA.Serialisation.Generators ()
 import Test.Tasty
 import Test.Tasty.QuickCheck
 
-trippingF ::
-  (Eq src, Show src, Show target, ToCBOR src) =>
-  (src -> Either target (BSL.ByteString, src)) ->
-  src ->
-  Property
-trippingF f x = runIdentity $ trippingM f' (===) x
-  where
-    f' a = pure (f a, toLazyByteString $ toCBOR a)
+-- trippingF ::
+--   (Eq src, Show src, Show target, ToCBOR src) =>
+--   (src -> Either target (BSL.ByteString, src)) ->
+--   src ->
+--   Property
+-- trippingF f x = runIdentity $ trippingM f' (===) x
+--   where
+--     f' a = pure (f a, toLazyByteString $ toCBOR a)
```

Comment:
The reason behind leaving these comments here is not clear because there is no mention. May be a special comment at the top will make it clear.